Unix/configeditor/configeditor.cpp

```diff
@@ -316,6 +328,39 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         Ftprintf(stderr, USAGE, arg0);
         exit(1);
     }
+
+    //For Reconfigure setting dynamically
+    if(opts.reconfig)
+    {
+        status = PIDFile_Read(&pid);
+        if (!status && pid != -1)
+        {
+            if(kill(pid, SIGUSR2) == 0)
+            {
+                //Send signal to all omiagents
+                #if defined(linux)
+                    result = system("ps -A -o pid,comm | awk '$2~/omiagent/{print $1}' | xargs -r kill -s SIGUSR2");
+                #elif defined(sun)
+                    result = system("ps -A -o pid,comm | awk '$2~/omiagent/{print $1}' | xargs kill -s USR2");
+                #else
+                    result = system("ps -A -o pid,comm | awk '$2~/omiagent/{print $1}' | xargs kill -s SIGUSR2");
+                #endif
+                if(result == 0)
+                {
+                    Ftprintf(stderr, ZT("Settings has been applied Dynamically\n"));
```

Comment:
`cout << "Settings has been applied Dynamically\n" << std::endl;` ?

Reply:
Changed to cout